.github/workflows/snap.yaml

```diff
@@ -0,0 +1,14 @@
+name: Snap
+
+on:
+  push: {}
+  pull_request: {}
+  workflow_dispatch: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
```

Comment:
We will probably want to use one of the project's runners. I'll let @mitchellh give a better review here.